src/biokbase/narrative/tests/generate_test_results.py

```diff
@@ -16,9 +16,13 @@
 )
 
 """
-generate_test_results.py is used to generate the job message data that the narrative backend produces and that the frontend consumes. It uses data from `ee2_job_test_data_file` and `app_specs_file` and provides expected narrative backend message data, as well as a number of mappings that are used in python tests.
+generate_test_results.py is used to generate the job message data that the narrative backend produces and that the frontend consumes.
```

Comment:
Introduced line breaks since my IDE is not set to line wrap

Reply:
What?! Which IDE are you using? Do you just not have line wrapping turned on?

Reply:
VSCode. So I can see when I'm being naughty and going too long?